packages/gasket-plugin-https/lib/actions.js

```diff
@@ -5,14 +5,25 @@ const one = require('one-time/async');
 const errs = require('errs');
 const { getPortFallback, portInUseError, startProxy, getRawServerConfig } = require('./utils');
 
+/**
+ * Gasket action: prepareServer
+ * @param {import('@gasket/core').Gasket} gasket Gasket instance
+ * @returns {Promise<void>} promise
+ * @public
+ */
+async function prepareServer(gasket) {
+  await gasket.isReady;
+  await gasket.exec('preboot');
+}
+
 /**
  * Gasket action: startServer
  * @param {import('@gasket/core').Gasket} gasket Gasket instance
  * @returns {Promise<void>} promise
  * @public
  */
 async function startServer(gasket) {
-  await gasket.isReady;
+  await gasket.actions.prepareServer();
```

Comment:
See other comment - removed the unnecessary action, execute `preboot` directly with a distinguishable name.

```suggestion
  await gasket.isReady;
  await gasket.exec('preboot');
```